CHANGES.md

```diff
@@ -1,3 +1,28 @@
+## stable-2.3.1
+This stable release adds a number of proxy stability improvements.
+
+To install this release, run: `curl https://run.linkerd.io/install | sh`
+
+**Special thanks to**: @zaharidichev and @11Takanori!
+
+**Full release notes**:
+* Proxy
+  * Changed the proxy's routing behavior so that, when the control plane
+    does not resolve a destination, the proxy forwards request with minimal
+    additional routing logic
+  * Fixed a bug in the proxy's HPACK codec that could cause requests with
+    very large header values to hang indefinitely.
+  * Replaced the fixed reconnect backoff with an exponential one (thanks,
+      @zaharidichev!)
+  * Fixed an issue where load balancers can become stuck
+  * Added a dispatch timeout that limits the amount of time a request can be
+    buffered in the proxy
+  * Removed the limit on the number of concurrently active service discovery
+    queries to the Destination service
```

Comment:
total nit, but Capitalization of "Destination service" seems strange, maybe either both caps or neither? If that is indeed correct, feel free to ignore.

Reply:
I was curious about this too as I'm drafting edge release notes. Right now there are 3 "Destination service" mentions and 3 "destination service" mentions so Dennis I guess you are the tie-breaker? :)

Reply:
This is too much pressure haha. No caps feels much better?